src/Actions/Family/CreateFamily.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Choredo\Actions\Family;
+
+use Assert\Assertion;
+use Choredo\Hydrators;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Teapot\StatusCode;
+use Zend\Diactoros\Response\JsonResponse;
+
+class CreateFamily
+{
+    /**
+     * @var Hydrators\Family
+     */
+    private $familyHydrator;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * CreateFamily constructor.
+     * @param Hydrators\Family $familyHydrator
+     * @param EntityManagerInterface $entityManager
+     */
+    public function __construct(Hydrators\Family $familyHydrator, EntityManagerInterface $entityManager)
+    {
+        $this->familyHydrator = $familyHydrator;
+        $this->entityManager = $entityManager;
+    }
+
+    public function __invoke(ServerRequestInterface $request, ResponseInterface $response)
+    {
+        $body = $request->getBody()->getContents();
+        Assertion::isJsonString($body);
+
+        $parsedBody = json_decode($body, true);
+
+        $data = $parsedBody['attributes'] ?? [];
```

Comment:
If we're using JSONApi shouldn't requests be coming in inside a master `data` key?